easybuild/easyconfigs/s/SPAdes/spades.patch

```diff
@@ -0,0 +1,58 @@
+* #import is unknown pragma for icpc, but apparently it can be removed without problems...
+* translating 'a < b' to 'b > a' is required to dance around weird compiler error: error: expected a ">"
+* __is_trivial define is required to dance around bug in Intel compilers (intel/2016b), see https://software.intel.com/en-us/forums/intel-c-compiler/topic/685388
+* cast to bool is required because Intel compilers complaining there's no implicit conversion to bool when Boost 1.63.0 is used as dependency
+--- SPAdes-3.10.1/src/common/modules/path_extend/ideal_pair_info.hpp.orig	2017-03-02 20:23:08.979343521 +0100
++++ SPAdes-3.10.1/src/common/modules/path_extend/ideal_pair_info.hpp	2017-03-02 20:23:26.519525771 +0100
+@@ -14,7 +14,6 @@
+ 
+ #ifndef IDEAL_PAIR_INFO_HPP_
+ #define IDEAL_PAIR_INFO_HPP_
+-#import <vector>
```

Comment:
I would prefer to change them to `#include`. This probably works because the import/include it multiple times but may break in the future.